  <launch>
  	<param name="steering_factor" value="18" />
  	<param name="rear_wheels_baseline" value="1.3" />
  	<param name="distance_front_rear" value="1.765" />
  	<node pkg="my_car" type="car_listener" name="car_listener" output="screen"/>
    
  </launch>
